spring-jasper-example/src/main/resources/application-bill-local.yml

```
@@ -0,0 +1,7 @@
+server:
+  port: ${SERVER_PORT:8083}
+
+spring:
+
+  application:
+    name: spring-jasper-example-bill-local
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)